imx-bootlets: Fix paths for used during boot stream generation

This will be run during image generation thus this won't use a
complete path to allow for relocation.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

diff --git a/linux.bd b/linux.bd
index 516f4f2..366ac22 100644
--- a/linux.bd
+++ b/linux.bd
@@ -4,10 +4,10 @@
 	flags = 0x01;
 }
 sources {
-	power_prep="./power_prep/power_prep";
-	sdram_prep="./boot_prep/boot_prep";
-	linux_prep="./linux_prep/output-target/linux_prep";
-	zImage = "./zImage";
+	power_prep="imx-bootlets-power_prep-@MACHINE@";
+	sdram_prep="imx-bootlets-boot_prep-@MACHINE@";
+	linux_prep="imx-bootlets-linux_prep-@MACHINE@";
+	zImage="zImage-@MACHINE@.bin@DTB@";
 }
 
 section (0) {
